src/ClusterBootstrap/deploy.py

```diff
@@ -1670,11 +1670,6 @@ def deploy_webUI_on_node(ipAddress):
     utils.sudo_scp(config["ssh_cert"], "./deploy/RestfulAPI/config.yaml",
                    "/etc/RestfulAPI/config.yaml", sshUser, webUIIP)
 
-    utils.render_template_directory(
```

Comment:
We still need to integrate dashboard deployment into the cloud-init deployment pipeline somewhere. Can you sync with @YinYangOfDao on this?

Reply:
sure. should I submit another PR to integrate this into cloud-init pipeline? or collaborate with Qixiang and do it in this PR?

Reply:
Continued in this PR